string user
string password
string path
---
uint32 map_id
float64 max_utm_x
float64 min_utm_x
float64 max_utm_y
float64 min_utm_y
float64 tf_x
float64 tf_y
float64 tf_yaw
string path
